### PR TITLE
NEW REG test case using new ID: 12.

### PR DESCRIPTION
**Pull request by harinineelu**
Also includes the generated config file.

### CODE REVIEW

File: src/harness/testcases/WINNF_FT_S_REG_testcase.py [hunk 1]
```diff
@@ -743,7 +743,106 @@ def test_WINNF_FT_S_REG_11(self, config_filename):
...
+    # Device_b is Category B with conditionals pre-loaded.
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam']
+    }
+    conditionals = {'registrationData': [conditionals_b]}
```
Comment (jni2000) [inline src/harness/testcases/WINNF_FT_S_REG_testcase.py]: Should we remove the 'conditional' from device_b registration request as it is preloaded?

Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_REG_testcase.py]: Done

Comment (harinineelu) [reply src/harness/testcases/WINNF_FT_S_REG_testcase.py]: Done

**Comment by harinineelu** [comment]
Thanks for the review, Deleted conditionals for device_b. 
Could you please take a look.

**Comment by harinineelu** [comment]
Merged #400

**Pull request merged**